docs/src/pages/components/cards/cards.md

```diff
@@ -47,6 +47,14 @@ By default, we use the combination of a `<div>` element and a _background image_
 
 ## UI Controls
 
+Often cards allow users to interact with the entirety of their surface to trigger its main action, be it an expansion, a link to another screen or some other behavior. The action area of the card can be specified by wrapping its contents in a `CardActionArea` component.
+
+{{"demo": "pages/components/cards/ActionAreaCard.js", "bg": true}}
+
+A card can also offer supplemental actions which should stand detached from the main action area in order to avoid event overlap. This is usually achieved by vertically separating the two Action Areas in order to keep a clear visual hierarchy between them.
```

Comment:
```suggestion
A card can also offer supplemental actions which should stand detached from the main action area in order to avoid event overlap.
```
Deleting the second sentence since: there's only one ActionArea, and the visual hierarchy is not the same as wrapping the whole lot in a single action area. :)

Reply:
Makes sense, didn't really know how to word it as I was referencing material.io where the second vertical chunk with the buttons is also referred to as an "action area".
Still, I definitely agree and its much more understandable like this. Less is more in this case :)